packages/server/lib/plugins/index.js

```diff
@@ -91,6 +92,14 @@ const init = (config, options) => {
     }
 
     debug('forking to run %s', childIndexFilename)
+
+    if (inspector.url()) {
+      childOptions.execArgv = _.chain(process.execArgv.slice(0))
+      .remove('--inspect-brk')
+      .push(`--inspect=${process.debugPort + 1}`)
```

Comment:
i guess this kinda addresses https://github.com/cypress-io/cypress/issues/5802 ?